CHANGELOG.md

```diff
@@ -0,0 +1,72 @@
+# API changes in Preview 3:
+
+🆕 New `ObservableValidator` class, which supports the [`INotifyDataErrorInfo`](https://docs.microsoft.com/dotnet/api/system.componentmodel.inotifydataerrorinfo) interface.
+
+🆕 Added new constructors with cancellation support to the async commands, and added new cancellation-related properties to the async command interfaces.
+
+🆕 Added a new `WeakReferenceMessenger` type. This type is less performant than the other messenger, and uses more memory, and in return only uses weak references to track recipients. This type essentially mirrors the behavior of the `Messenger` type from `MvvmLight`, making the transition easier for developers migrating from that library.
+
+🆕 Introduced a new custom delegate to represent message handlers, which also receives the current recipient as additional input parameter (see code changes below).
+
+✅ Renamed `Messenger` to `StrongReferenceMessenger`.
+
+✅ The `WeakReferenceMessenger` is now the default messenger used by the `ObservableRecipient` class.
+
+✅ Changed `ObservableObject` overloads using `Expression<Func<T>>` to be more efficient (see code changes below).
+
+✅ API changes to the `SetPropertyAndNotifyOnCompletion` (as detailed in [this blog post]( https://devblogs.microsoft.com/pax-windows/mvvm-toolkit-preview-3-the-journey-of-an-api/)).
```

Comment:
This and the IoC one need to be in the section below as well. 🙂

Reply:
Added more info in https://github.com/windows-toolkit/MVVM-Samples/pull/23/commits/5244c220822add0d95d35d78149aaeff30833765 and also linked to the preview docs in my other branch about the migration for the `Ioc` class 😄